src/azure-cli/azure/cli/command_modules/appservice/_validators.py

```diff
@@ -219,6 +219,9 @@ def _validate_service_tag_format(cmd, namespace):
         name = namespace.name
         webapp = _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'get')
         service_tag_full_list = network_client.service_tags.list(webapp.location).values
```

Comment:
curious, is it still worth doing this additional validation check here then? Are there other scenarios where this will provide value? If we are skipping this during failures?

Reply:
It is relevant. This will just unblock the few scenarios where users do not have permissions to pull the list of tags, but still help all users with permissions.